### DISCUSSION TITLE
2024/02/home-to-house/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/xiangyun#101] [General]
# 2024/02/home-to-house/

https://xiangyun.rbind.io/2024/02/home-to-house/

**Comment by XiangyunHuang** [comment]
昨天，我刚看完王小波的《我的精神家园》，没想到我春节写的这篇文章与之如此相合，当然，人家写得比我好多了。30 年过去了，人们还是老样子，我行我素，没有人会因为某个人写了一篇文章，讲了几句话而有所改变。读文学读出了一种挣扎过后的无力感。

**Reply by XiangyunHuang** [reply]
今天恰巧又看到阎连科10年前在香港科技大学的一个讲座[在大陆写作的特殊性](https://www.bilibili.com/video/BV1EC4y1o7r1/)，其中一个话题也是乡村变化，真是如此凑巧。

**Reply by XiangyunHuang** [reply]
讲座中，他也说王小波的杂文写的很好。我刚开始看的第一本杂文集是吴南星的《三家村札记》，[以为杂文是那样的](https://xiangyun.rbind.io/2024/02/san-jia-cun/)，不过，若是现在的作家还是明史专家的写作风格，估计极不容易，受众更少，反而，王小波的更当代一些、更接地气一些。

**Reply by XiangyunHuang** [reply]
再又想，更应该是作者身处的时代不同，经历不同，同时受众也是。

**Comment by earfanfan** [comment]
呃，农村一直都挺黑暗的，人性之幽暗深邃复杂总是一言难尽的。有句俗语说得好，过去无法改变，未来由自己创造。

**Reply by XiangyunHuang** [reply]
这里不是说农村黑暗和复杂，没到那个程度，只是说大家相互攀比，「你盖两层半，我盖三层，你盖三层，我盖三层半」，这就导致房子之间相互遮挡。而原来村里的房子都是一层、两层的，房子的间距小，不会遮挡。

相互攀比带来两个表面问题，其一，房子相互遮挡光线，其二，从整村来说，大量房子外观显得庸俗、恶俗、低俗。这次由农民自发进行的乡村改造工程绝对不亚于大跃进时期对自然的改造。

浅层问题是村委不作为，明知道这种问题，却没有统一规划和设计（这可能需要几届村委班子监督执行，有行政资源成本），导致巨大的土地资源浪费。不作为的原因是干部们又有了一份收入来源。很多占田、占地的村民交罚款给村委及相关部门，盖一次房子会涉及各个部门。

深层问题是房地产行业的严重过剩，房价限跌，房价居高不下，有行无市，倒逼很多人在家建房，产能从城市转移到农村。

**Reply by earfanfan** [reply]
我写“农村一直都挺黑暗的”这句时，并不是想说你们村的，当时其实是联想到我们村的一些事情，比如三十年前村里打死人、多名村人长期强暴一名妇女、争抢耕地导致兄弟反目、邻居就界线问题互相争吵辱骂、重男轻女以及压榨妇女等等。

我读高二的时候是寝室的寝室长，当时就立规矩说禁止攀比，现在看待攀比的心态有些变化。人类是群居动物，群体里生存的人有强烈的认可需求，再加上农村社会主流价值观单一，大部分人都把人生价值观往同一个模板上套，一旦形成攀比的风气，不得不说，悲乎。

遮挡光线的问题是不是跟你们那地形高低不一，加上宅地基少也有关呢，遮挡光线和建房层数增加的关系也许可以反过来看。关于农村自建房的审美问题，我们村去年有在外面做生意赚了钱的人家回村建房子，据说是花了一百万，春节的时候我还去参观过，房屋构造和我们那其他人家的都很不一样，后来和小伙伴聊天得知这户人家的图纸是单独画的，而村里很多户人家房子的图纸都是通用的，我们那村子里很多房子用的图纸就那么几套，这才有大量的复制和相似。村里盖房子更多追求简单和实用吧。

关于侵占耕地盖房子的事，我们那有不少人家房屋外墙上贴了宣传单子，写禁止荒废耕地以及将耕地私自改成宅基地，没想到你们那里还可以，我还以为耕地保护哪哪都管得很严呢，看来我接触到的样本还是太少。让村委会对农民自建房做统一规划和设计，这难度很高，这样村委会对待农户还是管理的性质，而自从2006年取消农业税后，基层干部不用找农户收税，基层乡镇政府变得悬浮，村委会和农户的关系似乎也变得很微妙了。另，现在在推行“村民自治”，房屋采光权涉及的问题涉及整村人的利益，除非全村人共同面对，一起沟通这件事。

从城市转移到农村的产能比例应该很小，我倒觉得农村住房条件、生活环境等各方面问题存在已久，确实是需要改善的。根据最近的一些新闻来看，“房价限跌”的风向似乎已经转变了。我有点好奇“这次由农民自发进行的乡村改造工程绝对不亚于大跃进时期对自然的改造”，这个结论对应的具体论据是哪些，如果你愿意的话，可以给我展开讲讲吗？

最后，传统文化赋予“家”这个字太多正面的意向，我觉得家实际上应该是一个中性词，就像“情绪”这个词过去也被赋予太多负面的意向，情绪也该是一个中性词。与父母沟通太难了，我跟我爸很少讲话，但跟我妈还能说说，关于催婚的问题，唉，不得不说沟通起来真是痛苦得让人绝对不想回忆，现在能做到相处和谐是因为我使用了一些手段强行让我妈理解并做到了“课题分离”，我想了下这么多年能够坚持下来持续跟我妈各种沟通还是因为，信任她，相信她心疼我本人的动机会超过被脑中固化的价值观念驱使的那部分，当然这本身也会耗费极大量的心力去理解她的固有观念。从我的视角看，既要保护自己的心灵不受他人影响，也要尽量避开那些会伤害父母的方式，不然的话，和父母沟通本身已经是极其困难的考验了，将来要把碎裂的关系修补好更是考验。另外，有些事情其实也可以找朋友聊聊啊，我去年就深陷在我妈催婚的漩涡里，有好几次感觉自己要陷入下坠的抑郁深渊了，这种时候需要朋友拉自己一把，就有跟朋友讲一讲，当然最后逃离漩涡的关键还是在自己，不过有朋友拉一把的感觉真得蛮好的。

**Reply by earfanfan** [reply]
对了，跟 AI 聊聊也是好的，感觉 AI 的解题思路都好清晰。

**Reply by XiangyunHuang** [reply]
> 我有点好奇“这次由农民自发进行的乡村改造工程绝对不亚于大跃进时期对自然的改造”，这个结论对应的具体论据是哪些，如果你愿意的话，可以给我展开讲讲吗？

就是上面说的呀，沿公路两侧大量占农田建房子、原宅基地相互攀比（风格、高度、大门、院墙等），三俗是因为很多人没什么文化弄出来的房子很难看，「让他三尺又何妨」的是少数，看到的都是歪七扭八的。当然，我的样本也仅限于我及我去亲戚家沿途看到、听到的情况。有的地方，网红「刘伟元」的家乡都是统一规划的，还挺漂亮的。

最后，我没有抑郁，和父母意见不同很正常。经过实践和时间的证明，对一些重要的事，他们的判断都是错的，因为他们受视野所限、读书也少。目前，他们也基本不干涉了。他们的诉求，我文中已写，给他们老有所依，他们就不会缠着我了。就是要把利益诉求分清楚，日子就好过。就好比结婚这个事吧，当然是以我为主，我就不会听他们的。而他们养老要在农村搞自建房还是在县城里买房子，我告诉他们利弊，最后决定听他们的。总而言之，换位思考，首先考虑核心利益方在哪边，然后考虑相关利益方是否可以接受妥协。

**Reply by earfanfan** [reply]
>有的地方，网红「刘伟元」的家乡都是统一规划的，还挺漂亮的。

额，其实我之前在一个地方有悄悄说到湘云你应该也是秩序控，见[这里](https://github.com/earfanfan/yuanfan/pull/609)。按刘伟元和全村统一规划这两个关键词好像搜不到相关视频，回头你哪天有空找出来给俺看看呗。说到统一规划，我在网络上看到过一些不同的样本，一个是据说江浙有个小区全部居民同意整个推倒重建，另一个是政府曾经推行过的“合村并居”政策，前者是小区居民有钱并且重建利益一致，后者是政府推行政策，所以我还蛮好奇你提到的这个情况，于我而言约莫会是个新样本。

我原本好奇的其实是另一个点，我们那在人民公社时期对自然的改造是……是动员超多村民参与的，因为以前洪水泛滥，所以修闸修防洪大堤，还有填湖，以及治水方面的一些大工程，看到你写“不亚于”时脑子里自动带入的是我们村的情况，想象不到有撒乡村改造工程能调动那么大力量，于是才有此一问。

你告诉他们的利弊是撒？如果这个问题冒犯到你的话，你可以忽略。我的大伯在以前的年代读书读到了研究生，后来去了北京的一个大学当老师，在北京定居，我高考结束那年我大伯是想把我爷爷奶奶接过去的，不过我爷爷奶奶不愿意。我以前觉得适应城市生活很容易，而且城市里物资丰裕、医疗资源也多，应该是更适合养老的。不过前些天和一位博主讨论离岛流动的神明后，忽然明白（只是我的猜测）另一些，一是我爷爷一生种地对他的土地感情特别深厚，二是他在村里住了很多年，经历了农村从宗族社会时宗族集体变迁到今天的家庭模式，在浓郁的乡土情结中，村子才是他的根，是他已经完全适应的、有掌控感的，而去城市生活是未知的，不会像我以为的那么容易适应。

**Reply by XiangyunHuang** [reply]
> 额，其实我之前在一个地方有悄悄说到湘云你应该也是秩序控

你应该误解了，统一规划，统一的是外部景观，核心就是不能破坏原来的景观（人文景观和自然景观），不知你是否知道梁思成对北京城的规划以及他留下的建筑，乌镇也是，《权利的游戏》的主拍摄地君临城也是、杭州中国美术学院也是。回到刘伟元的家乡，他们那统一的是占地面积、房屋限高、房屋间距，这样道路通畅，至于具体到某一家，要整成什么样子，没有要求。所以人家那有海景、有山景、有旅游，我们这只能看人家的院墙、水泥墙。至于说，现在官方不允许占田，实际上看到的是大部分能占、该占的都占了。

> 你告诉他们的利弊是撒？

医疗、交通。他们也不看书、也没啥体育活动、抖音快手的使用比我时间长的多，以后回了家，他们在附近种点蔬菜、打打牌、扯扯淡。对于县城，具体到我老家，这是比地级市差不了多少的县级市，他们长期生活在城里，可不喜欢这城，不是不能适应，而是回到老家多舒服、多自在。什么种地感情特别深？什么城市生活未知？不是的。每个人情况不同，他们受不了原来农村的居住环境、生活条件和邻里关系，房子建好之后，现在这些都改善了。我还羡慕「采菊东篱下，悠然见南山」的生活呢。对了，我农村的家离县城不算太远，车程 20 -30分钟。

**Reply by earfanfan** [reply]
>我还羡慕「采菊东篱下，悠然见南山」的生活呢。

哈哈，讲真，我看到这的时候忍不住笑了。不过，鉴于咱俩交流的一些历史经验，互相误解的概率比较高，所以我要解释一下，并不是嘲笑你，大约是因为意想不到而觉得有点好笑。我对于陶渊明诗的一些背景是酱紫理解的，他曾经去做官，但是无法适应那种总得虚假社交的偏于功利的世俗生活，而且他的价值观也和整个官僚体系不相合，所以他追求的是天然自在、淡泊出世的生活。但湘云你给我的印象是积极入世的，所以你这么说让我感到诧异，然后就，莫名其妙地被戳中笑点了，请勿见怪。
